Sources/StreamChat/Models/Attachments/ChatMessageImageAttachment.swift

```diff
@@ -94,15 +118,12 @@ extension ImageAttachmentPayload: Decodable {
                 container.decodeIfPresent(String.self, forKey: .name)
         )?.trimmingCharacters(in: .whitespacesAndNewlines)
 
-        let previewUrl = try container.decodeIfPresent(URL.self, forKey: .thumbURL) ?? imageURL
```

Comment:
Did we decode this value from the payload? If yes, then this would be a breaking change.

Reply:
This does not exist, our API does not return any thumb_url for Images. Also, our ImageAttachmentPayload does not encode that value, so it would never be sent to the server even if has a customer you updated the previewUrl. So this is fine to remove